recipes/doxygen/doxygen_test_patch.diff

```diff
@@ -0,0 +1,14 @@
+diff --git a/testing/runtests.py b/testing/runtests.py
+index 83647c6..c50204f 100644
+--- a/testing/runtests.py
++++ b/testing/runtests.py
+@@ -176,6 +176,9 @@ def main():
+ 	parser.add_argument('--outputdir',nargs='?',default='.',help='output directory to write the doxygen output to')
+ 	args = parser.parse_args()
+ 
++	args.all = None
++	args.ids = [[i for i in range(1,66) if i!=12]]
```

Comment:
Tried 1 first, didn't work. Then printed out the variable, turns out it is 2 lists.